src/ext/BalExtension/wixext/Xsd/bal.xsd

```diff
@@ -195,6 +195,18 @@
         </xs:annotation>
     </xs:attribute>
 
+    <xs:attribute name="UseCorrectUserCulture" type="YesNoType">
+        <xs:annotation>
+            <xs:documentation>
+                When set to "yes", causes WixStdBA/Prereq BA to use the correct API for determining the user's culture settings. Otherwise, the previous API is used to maintain compatiblity with previous versions of WiX.
```

Comment:
Yes, I know, but if the bundle's running on XP/2003, it has different behavior. That's worth documenting.